lib/console/ai/vector/storable.ex

```diff
@@ -226,3 +226,35 @@ defimpl Console.AI.Vector.Storable, for: Console.Schema.Catalog.Mini do
 
   def prompt(%@for{}), do: ""
 end
+
+defimpl Console.AI.Vector.Storable, for: Console.Schema.Cluster.Mini do
+  alias Console.AI.Utils
+
+  def id(%@for{id: id}), do: id
+
+  def content(%@for{} = mini) do
+    """
+    A Kubernetes cluster with data like so:
+    name: #{mini.name}
+    handle: #{mini.handle}
+    distro: #{mini.distro}
+    version: #{mini.version}
+    current_version: #{mini.current_version}
+    kubelet_version: #{mini.kubelet_version}
+    health_score: #{mini.health_score}
+    node_count: #{mini.node_count}
+    pod_count: #{mini.pod_count}
+    namespace_count: #{mini.namespace_count}
+    availability_zones: #{mini.availability_zones}
+    runtime_services: #{mini.runtime_services}
```

Comment:
this is a list of structs and my guess is it won't stringify well unless it's json encoded